server/views/pages/bookAVisit/additionalSupport.njk

```diff
@@ -84,7 +54,7 @@
                   },
                   checked: formValues.additionalSupportRequired == "yes",
                   conditional: {
-                    html: supportOptionsHtml
+                    html: otherSupportHtml
```

Comment:
Check when showing it that the content is escaped (e.g. user enters: `<b>test</b>`)